lint.sh

```diff
@@ -78,17 +78,25 @@ check_bash(){
 }
 
 check_python(){
-  flake8 --exclude=.lintvenv . \
+  flake8 --exclude=.lintvenv,webhooktranslator . \
     && echo "Python syntax ok" \
     || { echo "Python syntax fail"; rc=1; }
 }
 
+check_webhooktranslator(){
+    pushd webhooktranslator
+    tox && echo "Webhook Translator Unit tests pass" \
+      || { echo "Webhook Translator Unit tests fail"; rc=1; }
+    popd
+}
```

Comment:
I considered the same thing, but I'm not sure how you define non python environments with tox?